scripts/completions/install.py

```diff
@@ -189,6 +202,101 @@ def _update_rc(
         assert mode == "uninstall"
 
 
+def _update_conda_scripts(
+    completions_dir: pathlib.Path,
+    mode: ConfigureMode,
+) -> None:
+    """Try to add a `source /.../completions/setup.{shell}` line automatically to conda's activation scripts.
+
+    Args:
+        completions_dir: Path to location of this script.
+        mode: Install or uninstall completions.
+    """
+
+    # Install or uninstall `source_line`.
+    source_lines = _get_source_script(completions_dir, None)
+
+    conda_path = pathlib.Path(os.environ["CONDA_PREFIX"])
+    activate_path = conda_path / "etc/conda/activate.d/nerfstudio-activate.sh"
+    if mode == "uninstall":
+        if activate_path.exists():
+            os.remove(activate_path)
+        CONSOLE.log(f":broom: Existing completions uninstalled from {conda_path}.")
+    elif mode == "install":
+        # Install completions.
+        with activate_path.open("w+", encoding="utf8") as f:
```

Comment:
When testing on my machine the directory didn't exist, I had to add a `activate_path.parent.mkdir(exist_ok=True, parents=True)` above this line

Reply:
Thanks for pointing it out. I didn't realise the directory could be missing.